airbyte-integrations/connectors/destination-oracle/src/main/java/io/airbyte/integrations/destination/oracle/OracleOperations.java

```diff
@@ -0,0 +1,205 @@
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.oracle;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.jdbc.SqlOperations;
+import io.airbyte.protocol.models.AirbyteRecordMessage;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import java.util.function.Supplier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OracleOperations implements SqlOperations {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(OracleOperations.class);
+
+  private String tablespace;
+
+  public OracleOperations(String tablespace) {
+    this.tablespace = tablespace;
+  }
+
+  @Override
+  public void createSchemaIfNotExists(JdbcDatabase database, String schemaName) throws Exception {
+    final String query = String.format(
+        "declare userExists integer; begin select count(*) into userExists from dba_users where upper(username) = upper('%s'); " +
+            "if (userExists = 0) then " +
+            "execute immediate 'create user %s identified by %s'; " +
+            "execute immediate 'alter user %s quota unlimited on %s'; " +
+            "end if; end;",
+        schemaName, schemaName, schemaName, schemaName, tablespace);
+    database.execute(query);
+  }
+
+  @Override
+  public void createTableIfNotExists(JdbcDatabase database, String schemaName, String tableName) throws Exception {
+    try {
+      if (!tableExists(database, schemaName, tableName)) {
+        database.execute(createTableQuery(schemaName, tableName));
+      }
+    } catch (Exception e) {
+      LOGGER.error("Error while creating table.", e);
+      throw e;
+    }
+  }
+
+  @Override
+  public String createTableQuery(String schemaName, String tableName) {
+    return String.format(
+        "CREATE TABLE %s.%s ( \n"
+            + "%s VARCHAR(64) PRIMARY KEY,\n"
+            + "%s NCLOB,\n"
+            + "%s TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP\n"
+            + ")",
+        schemaName, tableName,
+        JavaBaseConstants.COLUMN_NAME_AB_ID.substring(1), JavaBaseConstants.COLUMN_NAME_DATA.substring(1),
+        JavaBaseConstants.COLUMN_NAME_EMITTED_AT.substring(1),
+        JavaBaseConstants.COLUMN_NAME_DATA.substring(1));
+  }
+
+  /*
```

Comment:
No.  The `JavaBaseConstants` all start with underscores, which are toxic to Oracle for whatever reason.  (Underscores are just fine *within* identifiers, but if you try to start with one, it will blow up on you.)

Reply:
my question here was just referring to the commented out code. i'm asking if we need this commented out code.